stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/model/dto/UserDtos.kt

```diff
@@ -66,7 +78,7 @@ internal data class DownstreamUserDto(
 
 @JsonClass(generateAdapter = true)
 internal data class PartialUpdateUserDto(
-    val id: String,
-    val set: Map<String, Any>,
-    val unset: List<String>,
+    @Json(name = "id") val id: String,
```

Comment:
Do we need to add those annotations here?

Reply:
I think it was done because adding the annotation to all fields, but as we discussed in previous comment, we will keep it as it was